### PR TITLE
feat: enterprise language list

### PR DESCRIPTION
**Pull request by mitchell-phillips**

### CODE REVIEW

File: scripts/list-languages-in-enterprise.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                    slug
+                    name
+                  }
+                }
+            `);
+
+            hasNextPage = enterpriseInfo.enterprise.organizations.pageInfo.hasNextPage;
+            // workaround to avoid code duplication on next line, allows null initial value which graphql requires
+            // eslint-disable-next-line prefer-template, quotes
+            endCursor = '"' + enterpriseInfo.enterprise.organizations.pageInfo.endCursor + '"';
```
Comment (alexashley) [inline scripts/list-languages-in-enterprise.js]: I don't understand the eslint comment, isn't this templated version equivalent? 

```suggestion
            endCursor = `"${enterpriseInfo.enterprise.organizations.pageInfo.endCursor}"`;
```

```
> const a = null;
> '"' + a + '"' === `"${a}"`
true
```

File: README.md [hunk 2]
```diff
@@ -38,6 +38,21 @@ Example:
 $ node cli.js list-languages-in-organization -o OneGHEOrg
 ```
 
+### List Languages In Enterprise
+
+Given a GitHub Enterprise, find all Organizations in it, and list the languages used within each repository.
+
+_NOTE: Token used to run this option MUST have permission to list Orgs and Repos within for every Org in the Enterprise_
```
Comment (alexashley) [inline README.md]:
```suggestion
_NOTE: Token used to run this option MUST have permission to list Orgs and Repos for every Org in the Enterprise_
```

File: scripts/list-languages-in-enterprise.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        let endCursor = null;
+        const languagesInEnterprise = {};
+
+        while (hasNextPage) {
+            const enterpriseInfo = await graphql(`
+                query ListOrgs {
+                  viewer {
+                    login
+                  }
+                  enterprise(slug: "${argv.enterprise}") {
```
Comment (alexashley) [inline scripts/list-languages-in-enterprise.js:35]: I'm  not that familiar with graphql, but I wonder if this query could go further and also grab the language stats from the repos 

https://stackoverflow.com/a/70414784

File: scripts/list-languages-in-enterprise.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        },
+        percent: {
+            alias: "p",
+            default: 25,
+            describe: "what percentage of code is needed to qualify as a \"used\" language",
+            type: "number",
+        },
+    },
+    action: async (octokit, graphql, argv) => {
+        const organdrepo = [];
```
Comment (alexashley) [inline scripts/list-languages-in-enterprise.js]:
```suggestion
        const orgAndRepo = [];
```

Might as well camel case this like the other variables

File: scripts/list-languages-in-enterprise.js (hunk 3 above)
Comment (mitchell-phillips) [reply scripts/list-languages-in-enterprise.js:35]: We want to explore this more to see if we can reduce the number of API calls.  I'll create an issue and see if we can tackle this later

**Review by RobertKelly — APPROVED** [review]
👍🏻

**Pull request merged**